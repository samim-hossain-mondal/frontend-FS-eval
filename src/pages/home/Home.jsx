import React from 'react';
import Dashboard from '../../components/dashboard/Dashboard';
import './Home.css';

export default function Home() {
  return (
    <div className="HomePage">
      <Dashboard />
      {/* <h1>Home Page</h1>
      {
        localStorage.getItem('token') ? <h5>your token: {localStorage.getItem('token')}</h5> : null
      } */}
    </div>
  );
}