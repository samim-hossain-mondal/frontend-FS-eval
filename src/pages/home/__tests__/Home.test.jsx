import React from 'react';
import { render } from '@testing-library/react';
import Home from '../Home';

describe('Home', () => {
  it('should render the Home component', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});