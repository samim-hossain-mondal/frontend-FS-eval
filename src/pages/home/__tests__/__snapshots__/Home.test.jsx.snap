// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render the Home component 1`] = `
<div>
  <div
    class="HomePage"
  >
    <div
      class="dashboard-page"
    >
      <div
        class="container-left"
      >
        <div
          class="container-left-header"
        >
          <h1>
            CMS+
          </h1>
        </div>
        <div
          class="container-left-content"
        >
          <div
            class="dashboard-header"
          >
            <a>
              COLLECTION TYPES
            </a>
            <img
              alt="icon"
              src="icon-search-dark.png"
            />
          </div>
          <ul />
        </div>
        <div
          class="content-type-builder"
        >
          <h3>
            CONTENT TYPE BUILDER
          </h3>
        </div>
      </div>
      <div
        class="container-right"
      >
        <div
          class="content-type"
        >
          <div
            class="content-type-header"
          >
            <h2>
              Content Types
            </h2>
          </div>
          <div
            class="content-type-body"
          >
            <div
              class="content-type-body"
            >
              <div
                class="left"
              >
                <div
                  class="left-header"
                >
                  <a>
                    0
                     types
                  </a>
                  <img
                    alt="icon"
                    src="icon-search-dark.png"
                  />
                </div>
                <div
                  class="add-new-type-btn"
                >
                  <button>
                     + New Type 
                  </button>
                </div>
                <div
                  class="list-types"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
